Added info how to update existing environment to README. First try to… (#11)
echemdb/website: README.md

conda env create --force -f environment.yml
```

> Alternatively, if you want to update an existing environment use:
> 
> ```
> conda env update --name <your_env_name> --file environment.yml
> ```
> 
> 
> 

Reviewer: ```suggestion
Alternatively, if you want to install the required packages into an existing, environment use:

```
conda env update --name <your_env_name> --file environment.yml
```

```
